core/reflection.jvm/src/kotlin/reflect/jvm/internal/KCallableImpl.kt

```diff
@@ -112,65 +112,87 @@ internal abstract class KCallableImpl<out R> : KCallable<R>, KTypeParameterOwner
         return if (isAnnotationConstructor) callAnnotationConstructor(args) else callDefaultMethod(args, null)
     }
 
+    private val _parameterSizeWithContinuation = ReflectProperties.lazySoft {
```

Comment:
This feels like an overkill, I propose to inline it. :)

Reply:
Fixed in [8daf9c2](https://github.com/JetBrains/kotlin/pull/4840/commits/8daf9c24e12b5491bc35e6623e051f9c24520703).